common/constants.js

```diff
@@ -40,6 +40,7 @@ export const system = {
   black: "#2D2926",
   pitchBlack: "#0c0c0c",
   brand: "#2935ff",
+  link: "#0074D9",
```

Comment:
Nice color choice! I like this as a compliment to our brand blue, down to try it out and see how I feel.